io/io/inc/ROOT/RIoUring.hxx

```diff
@@ -56,6 +56,10 @@ public:
       static const bool available = RIoUring::CheckIsAvailable();
       return available;
    }
+
+   struct io_uring *raw() {
```

Comment:
`GetRing()`?

Reply:
@jblomer what do you think about `GetRawRing`? https://github.com/root-project/root/pull/6162/commits/02b3ccf12a8721363a2e7fdd867e0707ede16e30